--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/30] 115.200.54.155 () {44 vars in 740 bytes} [Fri Mar 22 15:19:05 2019] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/31] 115.200.54.155 () {46 vars in 742 bytes} [Fri Mar 22 15:19:05 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/32] 115.200.54.155 () {44 vars in 740 bytes} [Fri Mar 22 15:19:06 2019] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/33] 115.200.54.155 () {46 vars in 742 bytes} [Fri Mar 22 15:19:06 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/34] 115.200.54.155 () {44 vars in 740 bytes} [Fri Mar 22 15:19:06 2019] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/35] 115.200.54.155 () {46 vars in 742 bytes} [Fri Mar 22 15:19:06 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 22 15:22:36 2019] ***
compiled with version: 5.4.0 20160609 on 22 March 2019 06:50:28
os: Linux-4.4.0-85-generic #108-Ubuntu SMP Mon Jul 3 17:23:59 UTC 2017
nodename: iZuf6ae9zoitqtjvgoeu75Z
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/blog
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/blog
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7857
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (:8001).
bind(): Address already in use [core/socket.c line 769]
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/36] 115.200.54.155 () {44 vars in 740 bytes} [Fri Mar 22 15:22:41 2019] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/37] 115.200.54.155 () {46 vars in 742 bytes} [Fri Mar 22 15:22:41 2019] GET /favicon.ico => generated 21 bytes in 1 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/38] 115.200.54.155 () {44 vars in 740 bytes} [Fri Mar 22 15:22:42 2019] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 19779|app: -1|req: -1/39] 115.200.54.155 () {46 vars in 742 bytes} [Fri Mar 22 15:22:42 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
invalid request block size: 21573 (max 4096)...skip
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 22 15:47:38 2019] ***
compiled with version: 5.4.0 20160609 on 22 March 2019 06:50:28
os: Linux-4.4.0-85-generic #108-Ubuntu SMP Mon Jul 3 17:23:59 UTC 2017
nodename: iZuf6ae9zoitqtjvgoeu75Z
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/blog
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/blog
your processes number limit is 7857
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on :8010 fd 3
uwsgi socket 0 bound to TCP address 127.0.0.1:8001 fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x21803d0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file <PROJECT_NAME>/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 22275)
spawned uWSGI worker 1 (pid: 22276, cores: 2)
spawned uWSGI worker 2 (pid: 22277, cores: 2)
spawned uWSGI worker 3 (pid: 22278, cores: 2)
spawned uWSGI worker 4 (pid: 22279, cores: 2)
*** Stats server enabled on 127.0.0.1:9191 fd: 18 ***
spawned uWSGI http 1 (pid: 22280)
--- no python application found, check your startup logs for errors ---
[pid: 22276|app: -1|req: -1/1] 115.200.54.155 () {36 vars in 478 bytes} [Fri Mar 22 15:48:12 2019] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Fri Mar 22 15:51:06 2019 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Mar 22 15:51:06 2019 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Mar 22 15:51:06 2019 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Mar 22 15:51:06 2019 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Mar 22 15:51:06 2019 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Mar 22 15:51:06 2019 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Mar 22 15:51:06 2019 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Mar 22 15:51:06 2019 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 22 16:00:33 2019] ***
compiled with version: 5.4.0 20160609 on 22 March 2019 06:50:28
os: Linux-4.4.0-85-generic #108-Ubuntu SMP Mon Jul 3 17:23:59 UTC 2017
nodename: iZuf6ae9zoitqtjvgoeu75Z
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/blog
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/blog
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7857
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on :8002 fd 3
spawned uWSGI http 1 (pid: 22400)
uwsgi socket 0 bound to TCP address :8001 fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x12750f0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 291680 bytes (284 KB) for 4 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x12750f0 pid: 22399 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (pid: 22399, cores: 1)
spawned uWSGI worker 2 (pid: 22402, cores: 1)
spawned uWSGI worker 3 (pid: 22403, cores: 1)
spawned uWSGI worker 4 (pid: 22404, cores: 1)
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/base.py", line 111, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 491, in resolve
    for pattern in self.url_patterns:
  File "/usr/local/lib/python3.5/dist-packages/django/utils/functional.py", line 37, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 533, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.5/dist-packages/django/utils/functional.py", line 37, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 526, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "./Blog/urls.py", line 24, in <module>
    path('',include('website.urls'))
  File "/usr/local/lib/python3.5/dist-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "./website/urls.py", line 2, in <module>
    from website import views
  File "./website/views.py", line 7, in <module>
    from markdown import markdown
ImportError: No module named 'markdown'
[pid: 22399|app: 0|req: 1/1] 115.200.54.155 () {36 vars in 478 bytes} [Fri Mar 22 16:00:48 2019] GET / => generated 148683 bytes in 149 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/base.py", line 111, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 491, in resolve
    for pattern in self.url_patterns:
  File "/usr/local/lib/python3.5/dist-packages/django/utils/functional.py", line 37, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 533, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.5/dist-packages/django/utils/functional.py", line 37, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 526, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "./Blog/urls.py", line 24, in <module>
    path('',include('website.urls'))
  File "/usr/local/lib/python3.5/dist-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "./website/urls.py", line 2, in <module>
    from website import views
  File "./website/views.py", line 7, in <module>
    from markdown import markdown
ImportError: No module named 'markdown'
[pid: 22403|app: 0|req: 1/2] 115.200.54.155 () {36 vars in 649 bytes} [Fri Mar 22 16:01:16 2019] GET / => generated 148868 bytes in 99 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.5/dist-packages/django/utils/deprecation.py", line 90, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.5/dist-packages/django/middleware/common.py", line 52, in process_request
    if self.should_redirect_with_slash(request):
  File "/usr/local/lib/python3.5/dist-packages/django/middleware/common.py", line 70, in should_redirect_with_slash
    not is_valid_path(request.path_info, urlconf) and
  File "/usr/local/lib/python3.5/dist-packages/django/urls/base.py", line 157, in is_valid_path
    resolve(path, urlconf)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/base.py", line 24, in resolve
    return get_resolver(urlconf).resolve(path)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 491, in resolve
    for pattern in self.url_patterns:
  File "/usr/local/lib/python3.5/dist-packages/django/utils/functional.py", line 37, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 533, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.5/dist-packages/django/utils/functional.py", line 37, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.5/dist-packages/django/urls/resolvers.py", line 526, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "./Blog/urls.py", line 24, in <module>
    path('',include('website.urls'))
  File "/usr/local/lib/python3.5/dist-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "./website/urls.py", line 2, in <module>
    from website import views
  File "./website/views.py", line 7, in <module>
    from markdown import markdown
ImportError: No module named 'markdown'
[pid: 22402|app: 0|req: 1/3] 115.200.54.155 () {36 vars in 633 bytes} [Fri Mar 22 16:01:17 2019] GET /favicon.ico => generated 164511 bytes in 104 msecs (HTTP/1.1 500) 1 headers in 63 bytes (1 switches on core 0)
[pid: 22403|app: 0|req: 2/4] 115.200.54.155 () {38 vars in 680 bytes} [Fri Mar 22 16:01:34 2019] GET / => generated 5411 bytes in 173 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Not Found: /static/css/style.css
[pid: 22403|app: 0|req: 3/5] 115.200.54.155 () {36 vars in 630 bytes} [Fri Mar 22 16:01:34 2019] GET /static/css/style.css => generated 3010 bytes in 21 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/images/aboutme.jpg
[pid: 22403|app: 0|req: 4/6] 115.200.54.155 () {36 vars in 661 bytes} [Fri Mar 22 16:01:34 2019] GET /static/images/aboutme.jpg => generated 3025 bytes in 11 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/css/normalize.css
[pid: 22402|app: 0|req: 2/7] 115.200.54.155 () {36 vars in 638 bytes} [Fri Mar 22 16:01:34 2019] GET /static/css/normalize.css => generated 3022 bytes in 373 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/images/logo.jpg
[pid: 22399|app: 0|req: 2/8] 115.200.54.155 () {36 vars in 655 bytes} [Fri Mar 22 16:01:34 2019] GET /static/images/logo.jpg => generated 3016 bytes in 373 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/css/code.css
[pid: 22404|app: 0|req: 1/9] 115.200.54.155 () {36 vars in 628 bytes} [Fri Mar 22 16:01:34 2019] GET /static/css/code.css => generated 3007 bytes in 380 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 22403|app: 0|req: 5/10] 115.200.54.155 () {40 vars in 686 bytes} [Fri Mar 22 16:01:34 2019] GET /favicon.ico => generated 2983 bytes in 8 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
[pid: 22403|app: 0|req: 6/11] 115.200.54.155 () {42 vars in 709 bytes} [Fri Mar 22 16:02:01 2019] GET / => generated 5411 bytes in 22 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 22402|app: 0|req: 3/12] 115.200.54.155 () {42 vars in 689 bytes} [Fri Mar 22 16:02:01 2019] GET /favicon.ico => generated 2979 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
<QuerySet [<Article: Article object (1)>, <Article: Article object (2)>]>
[pid: 22404|app: 0|req: 2/13] 115.200.54.155 () {44 vars in 764 bytes} [Fri Mar 22 16:02:45 2019] GET /tag/test => generated 5411 bytes in 121 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Article: Article object (1)>, <Article: Article object (2)>]>
[pid: 22403|app: 0|req: 7/14] 115.200.54.155 () {44 vars in 772 bytes} [Fri Mar 22 16:02:49 2019] GET /tag/test => generated 5411 bytes in 20 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22402|app: 0|req: 4/15] 115.200.54.155 () {42 vars in 709 bytes} [Fri Mar 22 16:02:50 2019] GET / => generated 5411 bytes in 49 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22403|app: 0|req: 8/16] 115.200.54.155 () {46 vars in 762 bytes} [Fri Mar 22 16:02:56 2019] GET / => generated 5411 bytes in 17 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 22402|app: 0|req: 5/17] 115.200.54.155 () {46 vars in 742 bytes} [Fri Mar 22 16:02:56 2019] GET /favicon.ico => generated 2979 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 22403|app: 0|req: 9/18] 115.200.54.155 () {44 vars in 740 bytes} [Fri Mar 22 16:04:33 2019] GET / => generated 5411 bytes in 18 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22402|app: 0|req: 6/19] 115.200.54.155 () {44 vars in 740 bytes} [Fri Mar 22 16:04:41 2019] GET / => generated 5411 bytes in 16 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22403|app: 0|req: 10/20] 115.200.54.155 () {44 vars in 766 bytes} [Fri Mar 22 16:04:43 2019] GET /article/1 => generated 3630 bytes in 15 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 22 16:05:50 2019] ***
compiled with version: 5.4.0 20160609 on 22 March 2019 06:50:28
os: Linux-4.4.0-85-generic #108-Ubuntu SMP Mon Jul 3 17:23:59 UTC 2017
nodename: iZuf6ae9zoitqtjvgoeu75Z
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/blog
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/blog
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7857
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on :8002 fd 3
spawned uWSGI http 1 (pid: 22470)
uwsgi socket 0 bound to TCP address :8001 fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x8540f0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 291680 bytes (284 KB) for 4 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x8540f0 pid: 22469 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (pid: 22469, cores: 1)
spawned uWSGI worker 2 (pid: 22472, cores: 1)
spawned uWSGI worker 3 (pid: 22473, cores: 1)
spawned uWSGI worker 4 (pid: 22474, cores: 1)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22469|app: 0|req: 1/1] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:05:53 2019] GET /article/1 => generated 3630 bytes in 168 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22472|app: 0|req: 1/2] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:05:54 2019] GET /article/1 => generated 3630 bytes in 190 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22472|app: 0|req: 2/3] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:09:22 2019] GET /article/1 => generated 3630 bytes in 12 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22473|app: 0|req: 1/4] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:09:22 2019] GET /article/1 => generated 4480 bytes in 197 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22472|app: 0|req: 3/5] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:09:26 2019] GET /article/1 => generated 3630 bytes in 13 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22473|app: 0|req: 2/6] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:09:28 2019] GET /article/1 => generated 4480 bytes in 15 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22472|app: 0|req: 4/7] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:09:28 2019] GET /article/1 => generated 3630 bytes in 12 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22473|app: 0|req: 3/8] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:09:29 2019] GET /article/1 => generated 4480 bytes in 14 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22472|app: 0|req: 5/9] 115.200.54.155 () {48 vars in 819 bytes} [Fri Mar 22 16:09:31 2019] GET /article/1 => generated 3630 bytes in 12 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 22473|app: 0|req: 4/10] 115.200.54.155 () {46 vars in 751 bytes} [Fri Mar 22 16:09:31 2019] GET /favicon.ico => generated 2979 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22474|app: 0|req: 1/11] 115.200.54.155 () {46 vars in 797 bytes} [Fri Mar 22 16:09:34 2019] GET /article/1 => generated 4480 bytes in 187 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22472|app: 0|req: 6/12] 115.200.54.155 () {44 vars in 757 bytes} [Fri Mar 22 16:09:37 2019] GET / => generated 5411 bytes in 16 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22473|app: 0|req: 5/13] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:09:40 2019] GET / => generated 6261 bytes in 20 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22472|app: 0|req: 7/14] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:09:47 2019] GET / => generated 5411 bytes in 16 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22473|app: 0|req: 6/15] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:09:48 2019] GET / => generated 6261 bytes in 17 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22472|app: 0|req: 8/16] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:09:50 2019] GET / => generated 5411 bytes in 18 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22473|app: 0|req: 7/17] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:10:22 2019] GET / => generated 6261 bytes in 24 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22472|app: 0|req: 9/18] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:10:30 2019] GET / => generated 5411 bytes in 18 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar 22 16:10:47 2019] ***
compiled with version: 5.4.0 20160609 on 22 March 2019 06:50:28
os: Linux-4.4.0-85-generic #108-Ubuntu SMP Mon Jul 3 17:23:59 UTC 2017
nodename: iZuf6ae9zoitqtjvgoeu75Z
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/blog
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/blog
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7857
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on :8002 fd 3
spawned uWSGI http 1 (pid: 22579)
uwsgi socket 0 bound to TCP address :8001 fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1fb00f0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 291680 bytes (284 KB) for 4 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1fb00f0 pid: 22578 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (pid: 22578, cores: 1)
spawned uWSGI worker 2 (pid: 22582, cores: 1)
spawned uWSGI worker 3 (pid: 22583, cores: 1)
spawned uWSGI worker 4 (pid: 22584, cores: 1)
[pid: 22578|app: 0|req: 1/1] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:10:50 2019] GET / => generated 6261 bytes in 208 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 1/2] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:10:50 2019] GET / => generated 6261 bytes in 218 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 1/3] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:10:51 2019] GET / => generated 6261 bytes in 213 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 2/4] 115.200.54.155 () {48 vars in 810 bytes} [Fri Mar 22 16:10:54 2019] GET / => generated 6261 bytes in 21 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 22582|app: 0|req: 3/5] 115.200.54.155 () {46 vars in 742 bytes} [Fri Mar 22 16:10:54 2019] GET /favicon.ico => generated 2979 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 2/6] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:12:31 2019] GET / => generated 6261 bytes in 19 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (2)
website.Article.None
[pid: 22582|app: 0|req: 4/7] 115.200.54.155 () {44 vars in 766 bytes} [Fri Mar 22 16:12:34 2019] GET /article/2 => generated 3213 bytes in 15 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 3/8] 115.200.54.155 () {44 vars in 757 bytes} [Fri Mar 22 16:12:36 2019] GET / => generated 6261 bytes in 19 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 5/9] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:19:02 2019] GET / => generated 6261 bytes in 22 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 6/10] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:21:30 2019] GET / => generated 6261 bytes in 18 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 4/11] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:21:31 2019] GET / => generated 6261 bytes in 19 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 7/12] 115.200.54.155 () {46 vars in 788 bytes} [Fri Mar 22 16:38:35 2019] GET / => generated 6261 bytes in 18 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 5/13] 115.200.54.155 () {42 vars in 719 bytes} [Fri Mar 22 16:41:41 2019] GET /admin => generated 0 bytes in 1 msecs (HTTP/1.1 301) 3 headers in 112 bytes (1 switches on core 0)
[pid: 22583|app: 0|req: 1/14] 115.200.54.155 () {42 vars in 721 bytes} [Fri Mar 22 16:41:41 2019] GET /admin/ => generated 0 bytes in 126 msecs (HTTP/1.1 302) 7 headers in 265 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 8/15] 115.200.54.155 () {42 vars in 758 bytes} [Fri Mar 22 16:41:41 2019] GET /admin/login/?next=/admin/ => generated 1821 bytes in 31 msecs (HTTP/1.1 200) 7 headers in 395 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 6/16] 115.200.54.155 () {46 vars in 878 bytes} [Fri Mar 22 16:42:15 2019] GET /admin/login/?next=/admin/ => generated 1821 bytes in 39 msecs (HTTP/1.1 200) 7 headers in 395 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 9/17] 115.200.54.155 () {54 vars in 1096 bytes} [Fri Mar 22 16:42:26 2019] POST /admin/login/?next=/admin/ => generated 0 bytes in 176 msecs (HTTP/1.1 302) 9 headers in 559 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 7/18] 115.200.54.155 () {48 vars in 949 bytes} [Fri Mar 22 16:42:26 2019] GET /admin/ => generated 5593 bytes in 25 msecs (HTTP/1.1 200) 6 headers in 227 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 10/19] 115.200.54.155 () {44 vars in 842 bytes} [Fri Mar 22 16:43:52 2019] GET / => generated 6261 bytes in 18 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 8/20] 115.200.54.155 () {46 vars in 873 bytes} [Fri Mar 22 16:45:06 2019] GET / => generated 6262 bytes in 23 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 11/21] 115.200.54.155 () {44 vars in 842 bytes} [Fri Mar 22 17:43:29 2019] GET / => generated 6262 bytes in 26 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 9/22] 115.200.54.155 () {46 vars in 873 bytes} [Fri Mar 22 17:48:18 2019] GET / => generated 6262 bytes in 25 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
<QuerySet [<Tag: test>]>
Article object (1)
website.Article.None
[pid: 22582|app: 0|req: 12/23] 115.200.54.155 () {46 vars in 899 bytes} [Fri Mar 22 17:51:07 2019] GET /article/1 => generated 4481 bytes in 17 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 10/24] 115.200.54.155 () {46 vars in 890 bytes} [Fri Mar 22 17:51:09 2019] GET / => generated 6262 bytes in 24 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Invalid HTTP_HOST header: 'blog.incubator4.com'. You may need to add 'blog.incubator4.com' to ALLOWED_HOSTS.
Bad Request: /
[pid: 22582|app: 0|req: 13/25] 122.226.116.102 () {38 vars in 429 bytes} [Fri Mar 22 17:53:51 2019] HEAD / => generated 57686 bytes in 36 msecs (HTTP/1.1 400) 1 headers in 53 bytes (1 switches on core 0)
[pid: 22584|app: 0|req: 11/26] 40.70.213.111 () {36 vars in 408 bytes} [Fri Mar 22 18:37:25 2019] GET / => generated 6262 bytes in 18 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Invalid HTTP_HOST header: 'blog.incubator4.com'. You may need to add 'blog.incubator4.com' to ALLOWED_HOSTS.
Bad Request: /
[pid: 22582|app: 0|req: 14/27] 112.113.196.243 () {38 vars in 429 bytes} [Fri Mar 22 18:54:33 2019] HEAD / => generated 57686 bytes in 36 msecs (HTTP/1.1 400) 1 headers in 53 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 15/28] 115.200.54.155 () {44 vars in 842 bytes} [Fri Mar 22 19:42:31 2019] GET / => generated 6262 bytes in 35 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 16/29] 115.200.54.155 () {48 vars in 921 bytes} [Fri Mar 22 20:25:55 2019] GET / => generated 6262 bytes in 17 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Not Found: /cgi-bin/config.exp
[pid: 22584|app: 0|req: 12/30] 128.14.134.134 () {36 vars in 539 bytes} [Fri Mar 22 20:31:16 2019] GET /cgi-bin/config.exp => generated 3000 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 22582|app: 0|req: 17/31] 115.200.54.155 () {44 vars in 842 bytes} [Fri Mar 22 20:35:14 2019] GET / => generated 6262 bytes in 19 msecs (HTTP/1.1 200) 3 headers in 110 bytes (1 switches on core 0)
Invalid HTTP_HOST header: 'blog.incubator4.com'. You may need to add 'blog.incubator4.com' to ALLOWED_HOSTS.
Bad Request: /
[pid: 22582|app: 0|req: 18/32] 112.45.91.118 () {38 vars in 426 bytes} [Fri Mar 22 21:46:43 2019] HEAD / => generated 57683 bytes in 59 msecs (HTTP/1.1 400) 1 headers in 53 bytes (1 switches on core 0)
